manifests/params.pp

```diff
@@ -161,26 +164,28 @@
       $needs_initdb = pick($needs_initdb, $postgresql::globals::manage_package_repo == true)
       $service_name = pick($service_name, 'postgresql')
 
-      $client_package_name    = pick($client_package_name, "postgresql-client-${version}")
-      $server_package_name    = pick($server_package_name, "postgresql-${version}")
-      $contrib_package_name = pick($contrib_package_name, "postgresql-contrib-${version}")
+      $client_package_name     = pick($client_package_name, "postgresql-client-${version}")
+      $server_package_name     = pick($server_package_name, "postgresql-${version}")
+      # debian package postgresql-${version} provides virtual package postgresql-contrib-${version}
+      $contrib_package_name    = pick($contrib_package_name, "postgresql-contrib-${version}")
+      $contrib_virtual_package = pick($contrib_virtual_package, 'yes')
```

Comment:
Is `'yes'` a valid boolean? I'd expect `true`

Reply:
I tried your point here 6d9958a

this is what you meant?